examples/charts/file/palettes.py

```diff
@@ -26,7 +27,11 @@
                                  ("YlGnBu9", YlGnBu9), ("YlOrBr9", YlOrBr9),
                                  ("YlOrRd9", YlOrRd9), ("Inferno9", Inferno9),
                                  ("Magma9", Magma9), ("Plasma9", Plasma9),
-                                 ("Viridis9", Viridis9)])
+                                 ("Viridis9", Viridis9), ("Accent8", Accent8),
+                                 ("Dark8", Dark8), ("Paired9", Paired9),
```

Comment:
I think Dark is actually called Dark2 (though there doesn't seem to exist a Dark1): https://jiffyclub.github.io/palettable/colorbrewer/qualitative/#dark2_3